tutorials/rendering/viewports.rst

```diff
@@ -163,93 +159,91 @@ it using (for example):
+  .. image:: img/default_scene.webp
 
-The other three options are Unshaded, Overdraw, and Wireframe. Unshaded draws the scene
-without using lighting information so all the objects appear flatly colored the color of
-their albedo.
+-  **Debug Draw = Unshaded**: Unshaded draws the scene without using lighting information so all the objects appear flatly colored in their albedo color.
 
-.. image:: img/unshaded.png
+  .. image:: img/unshaded.webp
 
-*The same scene with Debug Draw set to Unshaded*
+-  **Debug Draw = Overdraw**: Overdraw draws the meshes semi-transparent with an additive blend so you can see how the meshes overlap.
 
-Overdraw draws the meshes semi-transparent with an additive blend so you can see how the meshes overlap.
+  .. image:: img/overdraw.webp
 
-.. image:: img/overdraw.png
+-  **Debug Draw = Wireframe**: Wireframe draws the scene using only the edges of triangles in the meshes.
 
-*The same scene with Debug Draw set to Overdraw*
-
-Lastly, Wireframe draws the scene using only the edges of triangles in the meshes.
+  .. image:: img/wireframe.webp
 
 .. note::
 
-    The effects of the Wireframe mode are only visible in the editor, not while the project is running.
+    The effects of the Debug Draw modes may vary when using the Compatibility renderer.
 
 Render target
 -------------
 
-When rendering to a :ref:`Viewport <class_Viewport>`, whatever is inside will not be
-visible in the scene editor. To display the contents, you have to draw the :ref:`Viewport's <class_Viewport>` :ref:`ViewportTexture <class_ViewportTexture>` somewhere.
+When rendering to a :ref:`SubViewport <class_SubViewport>`, whatever is inside will not be
+visible in the scene editor. To display the contents, you have to draw the SubViewport's :ref:`ViewportTexture <class_ViewportTexture>` somewhere.
 This can be requested via code using (for example):
 
 ::
 
     # This gives us the ViewportTexture.
-    var rtt = viewport.get_texture()
-    sprite.texture = rtt
+    var tex = viewport.get_texture()
+    sprite.texture = tex
 
 Or it can be assigned in the editor by selecting "New ViewportTexture"
 
-.. image:: img/texturemenu.png
+.. image:: img/texturemenu.webp
 
 and then selecting the :ref:`Viewport <class_Viewport>` you want to use.
 
-.. image:: img/texturepath.png
+.. image:: img/texturepath.webp
 
-Every frame, the :ref:`Viewport <class_Viewport>`'s texture is cleared away with the default clear color (or a transparent
-color if :ref:`Transparent Bg<class_Viewport_property_transparent_bg>` is set to ``true``). This can be changed by setting ``Clear Mode`` to Never or Next Frame.
+Every frame, the :ref:`Viewport's <class_Viewport>` texture is cleared away with the default clear color (or a transparent
+color if :ref:`Transparent Bg<class_Viewport_property_transparent_bg>` is set to ``true``). This can be changed by setting :ref:`Clear Mode<class_SubViewport_property_render_target_clear_mode>` to ``Never`` or ``Next Frame``.
```

Comment:
```suggestion
color if :ref:`Transparent BG<class_Viewport_property_transparent_bg>` is set to ``true``). This can be changed by setting :ref:`Clear Mode<class_SubViewport_property_render_target_clear_mode>` to ``Never`` or ``Next Frame``.
```